src/initial_read_module.f90

```diff
@@ -2299,6 +2299,19 @@ subroutine read_input(start, start_L, titles, vary_mu,&
     ! Variable temperature
     flag_variable_temperature = fdf_boolean('MD.VariableTemperature', .false.)
     md_variable_temperature_method = fdf_string(20, 'MD.VariableTemperatureMethod', 'linear')
+
+    ! Verify that a method for variable temperature is valid
+    if (flag_variable_temperature) then
+      ! At present, only linear evolution is supported
+      if (md_variable_temperature_method .ne. 'linear') then
```

Comment:
Can we please use `leqi` from `input_module` to compare strings? Something like `if(.not.leqi(md_variable_temperature_method(1:6),'linear')` would work I think and be more portable (in particular it will ignore case...)